package.json

```diff
@@ -34,5 +34,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.12",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/plugin-syntax-jsx": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": "^18.2.1",
```

Comment:
I suggest you add this dependency since it shows that your app compiled with errors. You can run `npm ls` to see a list of missing dependancies.
To fix the error below, run `npm install airbnb`

error it has `npm ERR! missing: eslint-config-airbnb@^18.2.1, required by maths-magician@0.1.0`